Core/clim-basic/recording.lisp

```diff
@@ -2141,7 +2141,7 @@ according to the flags RECORD and DRAW."
                 (setf (output-record-position record)
                       (values (max cx (+ cx x1))
                               (if height
-                                  (max cy (+ cy (- height (- y2 y1))))
+                                  (max cy (+ cy (- height (+ (abs y1)))))
```

Comment:
works for me (nice catch). Is there a reason why there is `(+ (abs y1))` and not simply `(abs y1)`? If not, please remove the unnecessary function call. After that you may merge it right away and claim a bounty. Thanks!